lib/ansible/module_utils/compat/paramiko.py

```diff
@@ -5,10 +5,15 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
+from cryptography.exceptions import InternalError
```

Comment:
Since cryptography isn't a dep for older paramiko versions, and we don't have a specific version requirement, we won't be able to use this.